config/owasp/suppressions.xml

```diff
@@ -25,6 +25,8 @@
     <cve>CVE-2023-24998</cve>
     <cve>CVE-2023-28708</cve>
     <cve>CVE-2023-20861</cve>
+    <cve>CVE-2023-20862</cve>
+    <cve>CVE-2023-20873</cve>
```

Comment:
i have upgraded spring version to fix this CVE. If Yogendra is OK, it can be removed.